src/dashboard/src/components/administration/forms.py

```diff
@@ -468,4 +469,4 @@ def add_choice(
 
     def save(self, config_path):
         with open(config_path, "w") as f:
-            f.write(etree.tostring(self.xml, pretty_print=True))
+            f.write(etree.tostring(self.xml, pretty_print=True, encoding="unicode"))
```

Comment:
NB. Because we're serializing to disk I think this needs to be a byte encoding `UTF-8` per https://lxml.de/parsing.html